src/components/home/vacations-card.tsx

```diff
@@ -48,10 +50,10 @@ const VacationsCard = () => {
     adminMode ? allVacationRequestStatusesAtom : vacationRequestStatusesAtom
   );
   const [loading, setLoading] = useState(false);
-  const [persons] = useAtom(personsAtom);
-  const loggedInPerson = persons.find(
-    (person: Person) =>
-      person.id === config.person.forecastUserIdOverride || person.keycloakId === userProfile?.id
+  const [users] = useAtom(usersAtom);
+  const loggedInPerson = users.find(
+    (person: User) =>
+      person.forecastId === config.person.forecastUserIdOverride || person.id === userProfile?.id
```

Comment:
forecastUserIdOverride is currently depricated so it should be removed from here. Same should be applied to all the other loggedInPerson that you are updating in this PR. See suggestion

Reply:
Hopefully with this change if you log loggedIn**User**   You will see your own user- please test this